app/visualisation_methods/morris/chart.js

```diff
@@ -0,0 +1,45 @@
+window.morris_main = function (i, series_to_plot, sorted_parameters_names) {
+	var chart = new Highcharts.Chart({
+		chart: {
+			renderTo: $('#morris_chart_'+ i + " .chart")[0],
+			type: 'bar'
+		},
+		title: {
+			text: ''
```

Comment:
Slowa kluczowe typu "morris" zostana zmienione na ogolne (np. "sensitivity") w ramach dodania kolejnej metody wizualizacji analizy wrazliwosci. Zostanie to zrobione wraz z zadaniem SCAL-1051.